src/pretty-dom.js

```diff
@@ -5,25 +5,20 @@ import {getDocument} from './helpers'
 import {getConfig} from './config'
 
 const shouldHighlight = () => {
-  let colors
+  // Try to safely parse env COLORS: We will default behavior if any step fails.
   try {
-    colors = JSON.parse(process?.env?.COLORS)
-  } catch (e) {
-    // If this throws, process?.env?.COLORS wasn't parsable. Since we only
-    // care about `true` or `false`, we can safely ignore the error.
+    const colors = process?.env?.COLORS
+    if (colors) {
+      const b = JSON.parse(colors)
+      if (typeof b === 'boolean') return b
+    }
+  } catch {
+    // Ignore (non-critical) - Make a defaulting choice below.
   }
 
-  if (typeof colors === 'boolean') {
-    // If `colors` is set explicitly (both `true` and `false`), use that value.
-    return colors
-  } else {
-    // If `colors` is not set, colorize if we're in node.
-    return (
-      typeof process !== 'undefined' &&
```

Comment:
This always needs to stay for environments that don't have `process` e.g. browsers: https://github.com/testing-library/dom-testing-library/pull/1323